packages/raccoon-city-ui/src/components/Main/ChessGrid/FlatSidebarInfo/FlatSidebarInfo.tsx

```diff
@@ -157,8 +160,12 @@ export function FlatSidebarInfo(props: FlatSidebarInfoProps) {
                     </Button>
                 </SendRequestContainer>
             )}
-            {/* @ts-ignore */}
-            {isModalOpen && <FlatSidebarModal flat={flat} close={setModalOpen} />}
+            {isModalOpen && props.match.path.split('/')[1] === 'public' && (
```

Comment:
seek for another solution

Reply:
e.g. 
![image](https://user-images.githubusercontent.com/7975065/89989969-594c6d80-dc8a-11ea-8dd0-4531350836b1.png)